src/main/scala/piconot/Empty.scala

```diff
@@ -0,0 +1,24 @@
+import piconot.internal._
+import picolib.semantics._
+
+object Empty extends PicoLang("resources/empty.txt") {
+    // State 0: move left
+    when state 0 at "**x*" go West become 0		// go all the way to the left
```

Comment:
I like that you included the word state in your language. It makes this easier to understand than the original picobot.